Packs/opswat-metadefender/Integrations/integration-opswat-metadefender-v2.yml

```diff
@@ -129,6 +129,12 @@ script:
         res = http_req(method='GET', url_suffix=f'file/{scan_id}')
         return res
 
+    def get_dbot_file_context(file_hash, dbotscore):
+
+        return [{'Indicator': file_hash, 'Type': 'hash', 'Vendor': 'OPSWAT', 'Score': dbotscore},
```

Comment:
Same here, we don't need Type: 'hash', remove it .
Leave just the file

Reply:
done